vtiger_connector_sales/models/res_company.py

```diff
@@ -150,32 +164,40 @@ def sync_vtiger_sale_Quotes(self):
             if result.get('success'):
                 self.update_existing_sale_Quotes(result)
```

Comment:
Check update_existing_sale_Quotes and update_existing_sale_order methods, both are having the exact same code